### DISCUSSION TITLE
Dev 2.0 vs. Stable 1.33 wiegand reader

### DISCUSSION THREAD

**Discussion by Knutnoh** [omersiar/esp-rfid#603] [Q&A]
Hello together

On my blue relay board, the stable v.1.33 should work with my Wiegand reader but the keypad shouldn't work. On dev 2.0 my Wiegand reader shouldn't work. Can you help me what I can do? I must have these option to unlock the door with an PIN code only an enter # . 

My reader can u see here: https://a.co/d/h9RS58i

It's wired with d0 on gpio 12 and d1 on gpio 13

Thx

**Comment by matjack1** [comment]
hey @Knutnoh with `dev` branch you can have the Wiegand in "pin code only" mode that should allow you to do what you need.

If you don't find a way to make it work, please share a config file so that I can try having a look at your configuration and see if there's anything wrong :)

**Reply by Knutnoh** [reply]
There is my config. It seems that there are any problems with wiegand. my Reader works fine with older Versions of esp-rfid. If i use the newest version on dev, my reader didn´t give anthing of data. m reader sends data with 26bits, can there be the problem?
{
  "command": "configfile",
  "network": {
    "bssid": "******",
    "ssid": "******",
    "wmode": 0,
    "hide": 0,
    "pswd": "******",
    "offtime": 0,
    "dhcp": 1,
    "ip": "",
    "subnet": "",
    "gateway": "",
    "dns": "",
    "apip": "192.168.4.1",
    "apsubnet": "255.255.255.0",
    "fallbackmode": "1"
  },
  "hardware": {
    "readertype": 1,
    "wgd0pin": 13,
    "wgd1pin": 12,
    "sspin": 0,
    "rfidgain": 32,
    "wifipin": 255,
    "rtype": 1,
    "ltype": 0,
    "rpin": 4,
    "rtime": 400,
    "doorname": "Door",
    "beeperpin": 255,
    "ledwaitingpin": 255,
    "openlockpin": 255,
    "doorbellpin": 255,
    "accessdeniedpin": 255,
    "useridstoragemode": "decimal",
    "requirepincodeafterrfid": false,
    "allowpincodeonly": true,
    "removeparitybits": false,
    "doorstatpin": 255,
    "maxOpenDoorTime": 0,
    "relay2": {
      "rtype": 1,
      "ltype": 0,
      "rpin": 4,
      "rtime": 400
    },
    "relay3": {
      "rtype": 1,
      "ltype": 0,
      "rpin": 4,
      "rtime": 400
    },
    "relay4": {
      "rtype": 1,
      "ltype": 0,
      "rpin": 4,
      "rtime": 400
    },
    "numrelays": 1
  },
  "general": {
    "hostnm": "esp-rfid-haustuer",
    "restart": 0,
    "pswd": "******",
    "openinghours": [
      "111111111111111111111111",
      "111111111111111111111111",
      "111111111111111111111111",
      "111111111111111111111111",
      "111111111111111111111111",
      "111111111111111111111111",
      "111111111111111111111111"
    ]
  },
  "mqtt": {
    "enabled": 1,
    "host": "192.168.2.128",
    "port": 1111,
    "topic": "esp-rfid-haustuer",
    "autotopic": false,
    "user": "",
    "pswd": "",
    "syncrate": "180",
    "mqttlog": 0,
    "mqttha": 0
  },
  "ntp": {
    "server": "pool.ntp.org",
    "interval": 30,
    "timezone": 0
  }
}

**Reply by matjack1** [reply]
hello @Knutnoh sorry for the delay here, but if you can try now with the latest release: https://github.com/esprfid/esp-rfid/releases/tag/2.0.0.rc0 it would be awesome. Thank you!
